extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -308,6 +308,11 @@ public function getWikiVariables() {
 			$wikiVariables['facebookAppId'] = $egFacebookAppId;
 		}
 
+		$wikiImages = (new WikiService())->getWikiImages( [$this->wg->CityId], 500 );
```

Comment:
Array definitions should have spaces, where indexing syntax does not (https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax)